File: PATTERNS/Pattern10/Pattern10.go
// A
// A B
// A B C
// A B C D
// A B C D E
// A B C D E F

package main

import "fmt"

func main() {
	var n int
	val := 64
	fmt.Printf("Enter the number:")
	fmt.Scanf("%d", &n)
	for i := 1; i <= n; i++ {
		for j := 1; j <= i; j++ {
			fmt.Printf("%c ", val+j)
		}
		fmt.Printf("\n")
	}
}

File: PATTERNS/Pattern10/Pattern10.c
// A 
// A B 
// A B C 
// A B C D 
// A B C D E 
// A B C D E F

#include<stdio.h>

void main(){
    int n;
    int val = 64;
    printf("Enter the number:");
    scanf("%d",&n);
    for (int i = 1; i <=n; i++) {
        for(int j=1;j<=i;j++){
            printf("%c ",val+j);
        }
        printf("\n");
    }
}